brokerlib.php

```diff
@@ -930,7 +930,7 @@ function bigbluebuttonbn_broker_set_guest_password($bigbluebuttonbn, $params) {
     } else {
         $password = rand(1, 999999);
     }
-    $DB->set_field('bigbluebuttonbn', 'guestpass', $password, ['id' => $bigbluebuttonbn->id]);
     $callbackresponsedata = json_encode($password);
```

Comment:
I wonder if we should keep the set field on a generated password. That way there wont be confusion around generating it, and thinking it is done, vs the workflow where you enter it manually and hit save. Clicking generate then save isnt always obvious UX

Reply:
A way to go about it would be to show the Save/Submit button only once a change has been made (since the form isn't too long). Also, this gives the ability for the user potentially to enter a custom password of their choosing if that's what they want down the line. 

It also depends on how often they change the access codes, whether they do this together with the access time expiry or not.

Reply:
I mean just save it on generate, so even if they generate and dont save the page, it will still be present. Any save action on the page will still have the submit and will do the DB save as well.

Reply:
Okay, I'll re-add the set_field call.